### DISCUSSION TITLE
Strange behaviour with rule scheduler

### DISCUSSION THREAD

**Discussion by CondormanFr** [jertel/elastalert2#263] [Q&A] [answered]
Hi,
I'm trying to make elasalert2 working. it works but with a strange behaviuor.
When I start the elasalert2 docker container, i have to wait 2 hours for the rules to be working...
I set the scheduler to run every minute : i saw the query 
`'http://localhost:9200/elastalert_status/_search?pretty&size=1000' `
running every minutes, it's look ok.
**But**
No entries in `elastalert_status_status` index.

and 2 hours later (it seems), i saw queries like 
`http://localhost:9200/elastalert_status_status/_search?pretty&_source_includes=endtime%2Crule_name&size=1`
with my rule_name as answer and i have entries in `elastalert_status_status` index.

I'm in France, does it could have a link with UTC +2 ?
How to have the rules to works immediately ?
Thanks

**Comment by jertel** [comment, marked as answer]
Can you share your rule config? Be sure to redact sensitive values.

**Reply by CondormanFr** [reply]
I found the issue.
Like most of my containers, I add a line to bind the localtime of the container with the localtime of the server with 
```
 volumes:
   - /etc/localtime:/etc/localtime:ro      
```
As soon as i remove it, it work normally.
(before, I had done my tests on windows, and i didn't have this line)
